ui/components/component-library/form-text-field/form-text-field.tsx

```diff
@@ -8,7 +8,7 @@ import {
   Size,
 } from '../../../helpers/constants/design-system';
 
-import Box from '../../ui/box/box';
```

Comment:
More should be being done in these files to convert to typescript, namely adding types for props and removing propTypes which are not necessary when the types are specified for the props